doc/release-notes/release-notes-bucash1.5.1.0-rc1.md

```diff
@@ -0,0 +1,99 @@
+Release Notes for Bitcoin Unlimited Cash Edition 1.5.1.0-rc1
+=========================================================
+
+Bitcoin Unlimited Cash Edition release candidate 1 for version 1.5.1.0 is now available from:
+
+  <https://bitcoinunlimited.info/download>
+
+Please report bugs using the issue tracker at github:
+
+  <https://github.com/BitcoinUnlimited/BitcoinUnlimited/issues>
+
+This is a main release candidate based of Bitcoin Unlimited compatible
+with the Bitcoin Cash specifications you could find here:
+
+- https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/uahf-technical-spec.md (Aug 1st '17 Protocol Upgrade, bucash 1.1.0.0)
+- https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/nov-13-hardfork-spec.md (Nov 13th '17 Protocol Upgrade, bucash 1.1.2.0)
+- https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/may-2018-hardfork.md (May 15th '18 Protocol Upgrade, bucash 1.3.0.0, 1.3.0.1, 1.4.0.0)
+- https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/2018-nov-upgrade.md (Nov 15th '18 Protocol Upgrade, bucash 1.5.0.0, 1.5.0.1, 1.5.0.2)
+
+This release is compatible with both [Bitcoin Cash](https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/2018-nov-upgrade.md) and [SV](https://github.com/bitcoin-sv/bitcoin-sv/blob/master/doc/release-notes.md) changes to the consensus rules.
```

Comment:
I would reword this to 'This release candidate is compatible with ...' as well.

Reply:
fixed